test/test_other.py

```diff
@@ -10612,8 +10612,9 @@ def ok(args, filename='hello_world.cpp', expected='hello, world!'):
       self.run_process([EMCC, test_file(filename)] + args)
       self.assertContained(expected, self.run_js('a.out.js'))
 
-    # -O0 with BigInt support (to avoid the need for legalization)
-    required_flags = ['-sWASM_BIGINT']
+    # -O0 with BigInt support (to avoid the need for legalization) and STACK_OVERFLOW_CHECK=1 (to
+    # avoid the need for the stack check pass).
+    required_flags = ['-sWASM_BIGINT', '-sSTACK_OVERFLOW_CHECK=1']
```

Comment:
Hmm, this means the default build will still do wasm changes after link even after we default to wasm-bigint. Maybe it makes sense though.

Please add to this test. We want to emit a good error message in this case, that is, `ERROR_ON_WASM_CHANGES_AFTER_LINK` should do like it does with `WASM_BIGINT` - if it's not set right, suggest the proper value.